python/ray/tests/test_wait.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
```

Comment:
I think you'll need to add this to `python/ray/tests/BUILD` so that it runs in the CI

Reply:
Oh! I assumed (wrongly) that tests are automatically detected. Will fix.